### PR TITLE
DOMA-2218 fixed excessive table rerender in tickets view & changed the color of the tooltip to the same color of the tooltips in the table

### PR DESCRIPTION
**Pull request by Alllex202**

### CODE REVIEW

**Comment by toplenboren** [comment]
Its really hard to review if nothing is in description. Please add descripton or screenshots

File: apps/condo/domains/common/hooks/useMultipleFiltersModal.tsx [hunk 1]
```diff
@@ -49,6 +49,7 @@ import {
 } from '../utils/filters.utils'
 import { colors } from '../constants/style'
 import { Tooltip } from '../components/Tooltip'
+import { Ticket } from '../../../schema'
```
Comment (akarjakin) [inline apps/condo/domains/common/hooks/useMultipleFiltersModal.tsx:52]: this is strange

**Review by akarjakin — APPROVED** [review]

**Comment by Alllex202** [comment]
> Its really hard to review if nothing is in description. Please add descripton or screenshots

Before
https://user-images.githubusercontent.com/56914444/181483762-b7468579-0fb4-4b27-b8b1-630c224ef6db.mov

After

https://user-images.githubusercontent.com/56914444/181483891-850165ea-0355-419b-b341-35cd3da2e58a.mov

File: apps/condo/domains/common/hooks/useMultipleFiltersModal.tsx (hunk 1 above)
Comment (Alllex202) [reply apps/condo/domains/common/hooks/useMultipleFiltersModal.tsx]: its autocomplete)
i fixed it

**Review by VKislov — APPROVED** [review]

**Pull request merged**